x-pack/plugins/triggers_actions_ui/public/application/lib/alert_api.ts

```diff
@@ -126,7 +126,14 @@ export async function updateAlert({
   id: string;
 }): Promise<Alert> {
   return await http.put(`${BASE_ALERT_API_PATH}/${id}`, {
-    body: JSON.stringify(alert),
+    body: JSON.stringify({
+      throttle: alert.throttle,
+      name: alert.name,
+      tags: alert.tags,
+      schedule: alert.schedule,
+      params: alert.params,
+      actions: alert.actions,
+    }),
```

Comment:
nit.
I personally prefer using a helper function to do this kind of thing, like lodash's `pick`, just because I feel it's easier to read and maintain.
    
```suggestion
    body: JSON.stringify(pick(alert,[throttle, 'name', 'tags', 'schedule', 'params', 'actions'])),
```